src/view/droppable-dimension-publisher/droppable-dimension-publisher.jsx

```diff
@@ -72,16 +72,9 @@ export default class DroppableDimensionPublisher extends Component {
         return current;
       }
 
-      // We need to trim the dimension by the visible area of the scroll container
-
-      // Adjust the current dimension with the parents scroll
-      const top = current.top + scroll.y;
-      const bottom = current.bottom + scroll.y;
-      const left = current.left + scroll.x;
-      const right = current.right + scroll.x;
-
-      // Trim the dimension by the size of the parent
+      const { top, right, bottom, left } = current;
```

Comment:
It seems to honestly be as simple as this, but I'm hesitant to believe it's this easy... 🤔